library/options.json

```diff
@@ -185,6 +185,21 @@
       "type": "bool",
       "default_value": "false"
     },
+    "x_color": {
+      "type": "color",
+      "default_value": "0.841107, 0.16327, 0.120593",
```

Comment:
actually, I think we should not have a default value and, when not set, rely on F3Dstyle::GetF3DRed() instead.